statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
> this attitude is quite puzzling 

I don't want to offend you, or burn too much energy on this. But I'm pointing out a clear problem here, and I feel like you're simply ignoring my examples with "never seen it myself, so don't care". 

The actual code in this PR would simply error on a pandas dev version. I'd like pandas dev versions to work without a hitch - that's good for statsmodels users as well as for the quality of pandas releases.